arbitrum-docs/intro/intro.md

```diff
@@ -4,10 +4,10 @@ description: 'Arbitrum is a suite of Ethereum Layer-2 scaling solutions that you
 author: dzgoldman
 ---
 
-import PublicPreviewBannerPartial from '../partials/_public-preview-banner-partial.md';
+import PublicPreviewBannerPartial from '../partials/\_public-preview-banner-partial.md';
```

Comment:
I don't think we need that `\` character there, can you verify?

Reply:
Done, Thank you.